README.md

```diff
@@ -67,3 +67,8 @@ SELECT * FROM duckdb_functions() WHERE function_name LIKE 'cache_httpfs%';
 
 ## Design doc
 [doc](https://docs.google.com/document/d/1IU8NQ_1_u6g178DReO0LCKYdlAHfa1QRU25eRXhf4jI/edit?usp=sharing)
+
+## Benchmark 
+### [More About Benchmark](/benchmark/README.md)
+
+![sequential-read.cpp](benchmark-graph/seq-performance-barchart.svg)
```

Comment:
What do you mean by `SF`? I guess it's scaling factor, but no one knows what it is without context.

Reply:
scale factor is a widely used term, see duckdb [TPC-H extension doc](https://duckdb.org/docs/extensions/tpch.html)